patterns/2-structured/discover-your-innersource.md

```diff
@@ -52,27 +35,28 @@ Make it easy to find the reusable code.
 * Implement process change to first check for internal solutions for this problem.
 * Tool with a central view (but people are more inclined to google externally than look internally)
 * Concierge service (guide) to help product people find stuff. Might not scale but could be helpful in the beginning.
-* Need some very visible lighthouse projects that start using inner source components and make positive statements about the inner source program.
+* Need some very visible lighthouse projects that start using InnerSource components and make positive statements about the InnerSource program.
 * Establish a common, asynchronous communication channel (e.g., like slack or metamorph or yammer) across team boundaries. This might not scale beyond a certain organization size. It is possible people will start splitting this one channel into multiple channels by topic once traffic gets too high. Note: having one channel for many users of one tool might be considered an anti-pattern because they can't find it unless they already know about it.
 * Encourage (and reward) owners of reusable code to use the same search engine to continually search for products that are candidates for use and adoption of the reusable code but not currently doing so.
 * Consider creating a marketplace for marketing InnerSource programs (management can use this mechanism to know which InnerSource projects to fund, but seeing how the marketplace reacts).
 
-## Known Instances
-
-TBD
-
 ## Resulting Context
 
 * Internal components are easily visible
 * Developers looking for code can search for it and find it quickly.
 * Developers are now looking internally for software components
 * Search results are combined (internal and external)
 * Process changes, establishing a common communications channel, and encouraging and rewarding owners of reusable code to use the same search engine can contribute to changing the corporate culture. Transformation begins from the grassroots but requires strategic involvement of thought leaders.
-* See [Improved Findability](improve-findability.md) (aka Poor Naming Conventions or Badly Named Piles) as a related pattern.
+* See [Improved Findability](../1-initial/improve-findability.md) (aka Poor Naming Conventions or Badly Named Piles) as a related pattern.
```

Comment:
Add Findability to vale, or use discoverability?

https://github.com/InnerSourceCommons/isc-styles/blob/main/ISC/styles/config/vocabularies/ISC_General/accept.txt

Recommend using discoverability if we keep this link in this PR.

Reply:
I would opt for "discoverability".